package.json

```diff
@@ -70,8 +70,10 @@
     "request": "2.67.0",
     "serve-static": "1.10.0",
     "time-grunt": "1.2.1",
-    "ua-parser": "0.3.5",
-    "universal-analytics": "0.3.9"
+    "uap-core": "git://github.com/ua-parser/uap-core.git#1f509812",
+    "uap-ref-impl": "0.2.0",
+    "universal-analytics": "0.3.9",
+    "yamlparser": "0.0.2"
```

Comment:
@shane-tomlinson thanks for looking into fixing this up. this is a bit convoluted, a lot of custom logic below. It's also sad to see ua-parser not being maintained. 

We should really make a module out of this code update (something like https://github.com/fedot/node-uap) because we use ua-parser in at least one other servers (that also sends data to DataDog)